### PR TITLE
Update notebooks by re-syncing to `transformers` documentation

### PR DESCRIPTION
**Pull request by LysandreJik**

### CODE REVIEW

File: transformers_doc/pytorch/quicktour.ipynb [hunk 1]
```diff
@@ -78,7 +78,21 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pip install torch\n",
+    "pip install tensorflow\n",
+    "pip install transformers\n",
+    "pip install datasets"
```
Comment (philschmid) [inline transformers_doc/pytorch/quicktour.ipynb:95]:
```suggestion
    "#! pip install torch\n",
    "#! pip install tensorflow\n",
```

File: transformers_doc/quicktour.ipynb [hunk 2]
```diff
@@ -78,7 +78,21 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pip install torch\n",
+    "pip install tensorflow\n",
+    "pip install transformers\n",
+    "pip install datasets"
```
Comment (philschmid) [inline transformers_doc/quicktour.ipynb:95]:
```suggestion
    "#! pip install torch\n",
    "#! pip install tensorflow\n",
```

File: transformers_doc/tensorflow/quicktour.ipynb [hunk 3]
```diff
@@ -78,7 +78,21 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pip install torch\n",
+    "pip install tensorflow\n",
+    "pip install transformers\n",
+    "pip install datasets"
```
Comment (philschmid) [inline transformers_doc/tensorflow/quicktour.ipynb:95]:
```suggestion
    "#! pip install torch\n",
    "#! pip install tensorflow\n",
```

**Comment by ferreroxc** [comment]
Could you please have a look at this file notebooks/transformers_doc/custom_datasets.ipynb

The merge makes this file change to "How to fine-tune a model for common downstream tasks". Before the merge, it was "Fine-tuning with custom datasets". Why do you make this change in this notebook? I assume you could add a new notebook. I am very confused.

Also I have the same doubt for this page:
https://huggingface.co/docs/transformers/custom_datasets

Thank you.

**Pull request merged**